### PR TITLE
Add code for Closest Pair of Points Using Divide and Conquer Approach

### PR DESCRIPTION
**Pull request by pritamnegi**
#11

### CODE REVIEW

**Comment by pritamnegi** [comment]
Now take a look @ramitsawhney27

File: Algorithms/Divide-and-Conquer/closest_pair_of_points.cpp [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+// Function to store x and y coordinates into separate arrays
+// Then sorting the Px array according to x coordinate
+// and sorting the Py array according to y coordinate
+float closest(Point P[], int n)
+{
+    Point Px[n];
+    Point Py[n];
+
+    // Storing x and y coordinates in separate arrays
```
Comment (sahilbansal17) [inline Algorithms/Divide-and-Conquer/closest_pair_of_points.cpp]: Are you storing x and y coordinates separately or just making a copy of points array in the following loop?

**Comment by sahilbansal17** [comment]
@pritamnegi This code looks similar to that on geeksforgeeks. Please follow the [guidelines](https://github.com/sahilbansal17/Competitive_Coding/wiki/GSSoC#general-guidelines--expectations-from-you) strictly.

**Comment by pritamnegi** [comment]
Ok I will try to modify the code

File: Algorithms/Divide-and-Conquer/closest_pair_of_points.cpp [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+ 
+// Declaration of structure
+// for x any y coordinate of a point
+struct Point
+{
+    int x, y;
+};
+ 
+// Function to sort array of points according to x coordinate
+int compareX(const void* a, const void* b)
```
Comment (aditya81070) [inline Algorithms/Divide-and-Conquer/closest_pair_of_points.cpp]: @pritamnegi Completely copied from [GeeksforGeeks](https://www.geeksforgeeks.org/closest-pair-of-points/)

Comment (aditya81070) [reply Algorithms/Divide-and-Conquer/closest_pair_of_points.cpp]: @pritamnegi please change it

**Comment by pritamnegi** [comment]
Ok I will change the code

**Pull request closed without merging**